ert_shared/ensemble_evaluator/evaluator.py

```diff
@@ -90,19 +90,36 @@ async def _fm_handler(self, event):
 
     @_dispatch.register_event_handler(identifiers.EVTYPE_ENSEMBLE_STOPPED)
     async def _ensemble_stopped_handler(self, event):
-        snapshot_mutate_event = PartialSnapshot(self._snapshot).from_cloudevent(event)
-        await self._send_snapshot_update(snapshot_mutate_event)
+        if self._snapshot.get_status() != "Failure":
```

Comment:
What about if you de-register the handler in the event of a failure? 

So, in the case of a failure, you do this:
```python
# handle failure
_dispatch.deregister_event_handler(identifiers.EVTYPE_ENSEMBLE_STOPPED)
```
This is to make sure the ensemble behaves correctly. I.e. after an ensamble fails, it shouldn't start or be cancelled.

Reply:
Good points I will look into this.

Reply:
I leave it as is now. In principle no events should be emitted after the failure, so its more precaution, to make sure that the status is not changed after the fact, i.e. "Failure" should take precedence.

raising errors is still a problem, if you raise an error in a handler, the program hangs. Something that may bite us in the future, there is already an issue about that.